remote-api/src/test/java/org/alfresco/rest/api/impl/rules/RulesImplTest.java

```diff
@@ -611,4 +689,11 @@ private static List<Exception> ruleValidationExceptions()
             new RelationshipResourceNotFoundException(RULE_ID, "fake-relationship-id")
         );
     }
+
+    private void setNodeRefsForLinking()
+    {
+        given(nodeValidatorMock.validateFolderNode(eq(LINK_TO_NODE_ID), anyBoolean())).willReturn(linkToNodeRef);
+        given(nodeValidatorMock.validateFolderNode(eq(FOLDER_NODE_ID), anyBoolean())).willReturn(folderNodeRef);
```

Comment:
Could these go in the setUp method instead?  We could replace the existing `given(nodValidatorMock.validateFolderNode...` line.

Reply:
Oh - except that I suggested moving the method to `RuleSetImpl`, so all of this test code should go to `RuleSetImplTest` instead.